packages/database/package.json

```diff
@@ -34,8 +34,8 @@
     "lodash.keyby": "^4.6.0",
     "moment": "^2.24.0",
     "os": "0.1.1",
-    "pg": "7.10.0",
-    "pg-pubsub": "0.3.0",
+    "pg": "8.5.1",
+    "pg-pubsub": "0.6.1",
```

Comment:
Happy with this :+1:

It's the kind of change we'd want to merge soon after a release so that it gets a full week of people using it in dev, just in case there's any regression, because it's so central (particularly `pg-pubsub` - I haven't looked through those release notes and they're below v1 so could have breaking changes)

Reply:
I have put breaking changes and testing notes in the description